settings/common.py

```diff
@@ -62,6 +62,8 @@
     'rest_framework.authtoken',
     'rest_framework',
     'rest_framework_docs',
+    'rest_framework_expiring_authtoken',
+    'autofixture'
```

Comment:
`THIRD_PARTY_APPS` contains apps in alphabetical order except for `rest_framework.authtoken`.
Can you please add your apps in alphabetical order too?